File: Tyron2/src/main/scala/com/launcher/test/MainActivity.scala
package com.launcher.test

import android.app.Activity
import android.content.Context
import android.os.Bundle
import android.graphics.drawable.Animatable
import android.widget.Toast
import android.content.Intent
import com.example.myapplication.AndroidVideoCaptureExample



class MainActivity extends Activity {
    // allows accessing `.value` on TR.resource.constants
    implicit val context = this

    override def onCreate(savedInstanceState: Bundle): Unit = {
        super.onCreate(savedInstanceState)

        val text = "Hello toast!";
        val duration = Toast.LENGTH_SHORT;
        val toast = Toast.makeText(context, text, duration);
        toast.show();

        val intent = new Intent(this, classOf[AndroidVideoCaptureExanple]);
        println("tristeess");
    }

}

File: Tyron2/myapplication/src/main/java/com/example/myapplication/MainActivity.java
package com.example.myapplication;

import android.content.Intent;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.widget.Toast;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        //setContentView(R.layout.activity_main);
        //Intent intent = new Intent(this, AndroidVideoCaptureExample.class);
        //intent.addFlags(intent.FLAG_ACTIVITY_NEW_TASK);
        //startActivity(intent);
        Toast.makeText(this, "bonjour", Toast.LENGTH_LONG).show();
    }
}
